### PR TITLE
added stt_en_conformer_transducer_large_ls to NGC

### PR DESCRIPTION
**Pull request by VahidooX**
Signed-off-by: Vahid <vnoroozi@nvidia.com>

# What does this PR do ?

Added stt_en_conformer_transducer_large_ls  to NGC. A Conformer-Transducer-Large just trained on LS.

**Collection**: ASR

# Changelog 
- Added a new model stt_en_conformer_transducer_large_ls to NGC (A Conformer-Transducer-Large just trained on LS)

**PR Type**:
- [x ] New Feature
- [ ] Bugfix
- [ ] Documentation

### CODE REVIEW

File: nemo/collections/asr/models/asr_model.py [hunk 1]
```diff
@@ -11,16 +11,12 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-import logging
```
Comment (titu1994) [inline nemo/collections/asr/models/asr_model.py:14]: Why are these being removed?

File: nemo/collections/asr/models/asr_model.py [hunk 2]
```diff
@@ -11,16 +11,12 @@
...
-import os
-import re
 from abc import ABC, abstractmethod
 from typing import List, Optional, Union
 
 import torch
-from omegaconf import OmegaConf
 
-from nemo.core.classes import ModelPT, exportable, typecheck
+from nemo.core.classes import ModelPT
```
Comment (titu1994) [inline nemo/collections/asr/models/asr_model.py:19]: This is required for exporting no ?

**Review by titu1994 — COMMENTED** [review]
Some comments

File: nemo/collections/asr/models/asr_model.py (hunk 2 above)
Comment (VahidooX) [reply nemo/collections/asr/models/asr_model.py:19]: those imports were not being used.

File: nemo/collections/asr/models/asr_model.py (hunk 1 above)
Comment (VahidooX) [reply nemo/collections/asr/models/asr_model.py:14]: It has not been used here.

**Review by titu1994 — APPROVED** [review]
LGTM thanks !

**Pull request merged**